File: 1401-1500/1436/Solution.java
//简单的Hash表
class Solution {
    public String destCity(List<List<String>> paths) {
        HashSet<String> hSet = new HashSet<>();
        for(List<String> path : paths){
            hSet.add(path.get(0));
        }
        for(List<String> path : paths){
            if(!hSet.contains(path.get(1))) return path.get(1);
        }
        return "";
    }
}

File: 1401-1500/1436/solution.go
//简单的Hash表
func destCity(paths [][]string) string {
    dic := make(map[string]bool, 100)
    for _, v := range paths {
        dic[v[0]] = true
    }
    for _, v := range paths {
        if _, ok := dic[v[1]]; !ok {
            return v[1]
        }
    }
    return ""
}

